R/rdeephaven/prototyping/new-api.R

```diff
@@ -0,0 +1,27 @@
+Rcpp::compileAttributes()
+roxygen2::roxygenise()
+install.packages("/home/user/rdeephaven", repos=NULL, type="source")
+
+# load deephaven library
+library(rdeephaven)
+
+# connect to server using Client object
+client <- DeephavenClient$new(target="192.168.1.241:10000")
```

Comment:
Does this work for you?  If so, it is more portable to anyone running a server on the same machine.  We can address later.
```suggestion
client <- DeephavenClient$new(target="localhost:10000")
```

Reply:
I don't really plan on this file getting shipped. Should I get rid of it now?